src/test/java/org/jenkinsci/plugins/additionalmetrics/StdevDurationTest.java

```diff
@@ -0,0 +1,185 @@
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.jenkinsci.plugins.additionalmetrics;
+
+import com.gargoylesoftware.htmlunit.html.DomNode;
+import hudson.model.ListView;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.number.OrderingComparison.greaterThan;
+import static org.jenkinsci.plugins.additionalmetrics.PipelineDefinitions.failingDefinition;
+import static org.jenkinsci.plugins.additionalmetrics.PipelineDefinitions.sleepDefinition;
+import static org.jenkinsci.plugins.additionalmetrics.PipelineDefinitions.sleepThenFailDefinition;
+import static org.jenkinsci.plugins.additionalmetrics.PipelineDefinitions.slowDefinition;
+import static org.jenkinsci.plugins.additionalmetrics.PipelineDefinitions.unstableDefinition;
+import static org.jenkinsci.plugins.additionalmetrics.UIHelpers.createAndAddListView;
+import static org.jenkinsci.plugins.additionalmetrics.UIHelpers.dataOf;
+import static org.jenkinsci.plugins.additionalmetrics.UIHelpers.getListViewCell;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import java.util.OptionalDouble;
+import java.util.stream.Stream;
+
+
+public class StdevDurationTest {
+
+    @ClassRule
+    public static final JenkinsRule jenkinsRule = new JenkinsRule();
+
+    private StdevDurationColumn stdevDurationColumn;
+
+    @Before
+    public void before() {
+        stdevDurationColumn = new StdevDurationColumn();
+    }
+
+    @Test
+    public void no_runs_should_return_no_data() throws Exception {
+        WorkflowJob project = jenkinsRule.createProject(WorkflowJob.class, "ProjectWithZeroBuilds");
+
+        Duration stdevDuration = stdevDurationColumn.getStdevDuration(project);
+
+        assertNull(stdevDuration);
+    }
+
+    @Test
+    public void two_successful_runs_should_return_their_stdev_duration() throws Exception {
+        WorkflowJob project = jenkinsRule.createProject(WorkflowJob.class, "ProjectWithTwoSuccessfulBuilds");
+        project.setDefinition(sleepDefinition(1));
+        WorkflowRun run1 = project.scheduleBuild2(0).get();
+        project.setDefinition(sleepDefinition(6));
+        WorkflowRun run2 = project.scheduleBuild2(0).get();
+
+        Duration stdevDuration = stdevDurationColumn.getStdevDuration(project);
+        assertEquals(getStandardDeviation(Stream.of(run1.getDuration(), run2.getDuration()), Stream.of(run1.getDuration(), run2.getDuration()).mapToDouble(d -> d).average().getAsDouble()), stdevDuration.getAsLong());
```

Comment:
For readability reasons, may I suggest that you write this (and the others where relevant) as:

```
assertEquals(getStandardDeviationOf(Stream.of(run1.getDuration(), run2.getDuration()), stdevDuration.getAsLong());
```

Let `getStandardDeviationOf` do all the math. :-)

Also, might be interesting to use common-math in the tests to verify that the calculations are correct. What do you think?